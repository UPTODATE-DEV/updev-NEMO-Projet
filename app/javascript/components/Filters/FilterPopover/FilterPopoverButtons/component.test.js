import React from 'react';
import { shallow } from 'enzyme/build';

import Component from './component';

const defaultProps = {
  onSubmit: jest.fn(),
  containerClass: 'class',
};

it('renders as expected', () => {
  const wrapper = shallow(<Component {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
