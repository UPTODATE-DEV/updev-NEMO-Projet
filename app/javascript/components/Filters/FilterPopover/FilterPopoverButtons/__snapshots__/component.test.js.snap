// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<div
  className="btn-apply-container class"
>
  <Button
    className="btn-apply"
    onClick={[MockFunction]}
  >
    Apply
  </Button>
</div>
`;
